.cirrus.star

```diff
@@ -1,4 +1,4 @@
 load("github.com/SonarSource/cirrus-modules@v2", "load_features")
 
 def main(ctx):
-  return load_features(ctx)
+  return load_features(ctx, aws=dict(env_type="dev"))
```

Comment:
Remove this change when the docker images containing digicert tooling are promoted to prod

Reply:
done